lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -231,12 +235,36 @@ class NGPHistoryEvent {
   }
 }
 
-class UserTimeDateEvent extends NGPHistoryEvent {
+class TimeChangeEvent extends NGPHistoryEvent {
   get newTimestamp() {
     return NGPUtil.NGPTimestamp.fromBuffer(this.eventData.slice(0x0B, 0x13));
   }
 }
 
+class AlarmNotificationEvent extends NGPHistoryEvent {
```

Comment:
Do you actually make use of these alarm notification events anywhere?

Reply:
Not yet, but I have an outstanding card to support specific alarms (https://trello.com/c/PZfoCKuP). I did this work as part of the debugging to track down how best to solve this, but I've left it in so that I can complete that card later.